xml_inscriptions/bib_only/MI.AA.UM.KM.G.33632.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader><!-- testing1 -->
+      <fileDesc>
+         <titleStmt>
+            <title>MI.AA.UM.KM.G.33632</title>
+         </titleStmt>
+         <publicationStmt>
+            <xi:include  href="http://library.brown.edu/usep_data/resources/include_publicationStmt.xml">
+               <xi:fallback>
+                  <p>The Publications Statement should appear here. (c) Brown University</p>
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="MI.AA.UM.KM.G.33632">MI.AA.UM.KM.G.33632</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>MI</region>
+                  <settlement>AA</settlement>
+                  <institution>UM</institution>
+                  <repository>KM</repository>
+                  <idno>33632</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="grc" otherLangs=""/>
+                  <msItem class="#honorific">
+                     <p>fragmentary inscription naming deifed and living kings</p>
+                  </msItem>
+               </msContents>
+               <physDesc><!--objectDesc, handDesc, decoDesc go here-->
+                  <objectDesc ana="#stele">
+                     <supportDesc ana="#stone.other">
+                        <support>
+                           <p>broken limestone stele</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>25.2</height>
```

Comment:
If you want to make the dimensions a little more machine readable, you could add `@min` and `@max` attributes on the various dimensions (so, e.g. `<width min="16" max="20">16-20</width?>`)